AISKU/snippet/README.md

```diff
@@ -0,0 +1,48 @@
+<properties
+	pageTitle="Application Insights JavaScript SDK - Snippets"
+	description="Reference doc"
+	services="application-insights"
+    documentationCenter=".net"
+/>
+
+<tags
+	ms.service="application-insights"
+	ms.workload="tbd"
+	ms.tgt_pltfrm="ibiza"
+	ms.devlang="na"
+	ms.topic="article"
+	ms.date="8/28/2020"/>
+
+# Microsoft Application Insights JavaScript SDK - Snippet Injection
+
+## How to push Application Insights JS snippet changes consumed by Application Insights dotnet package.
+
+1.  Fork Application Insights dotnet [Repo](https://github.com/microsoft/ApplicationInsights-dotnet).
+2.  Get the latest copy of JS snippet and make changes in NETCORE\src\Microsoft.ApplicationInsights.AspNetCore\Resources.resx file.
```

Comment:
We need to modify the snippet to allow for instrumentation keys to be injected into the snippet from .NET

https://github.com/microsoft/ApplicationInsights-dotnet/issues/2031

Reply:
Spoke with @TimothyMothra offline. This seems like a no-op since the ikey isn't injected with `JsSnippet.format(ikey)` anymore

https://github.com/microsoft/ApplicationInsights-dotnet/blob/9b1431c35831a3f2c750603176b4436df5798363/NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/JavaScriptSnippet.cs#L116